scripts/calculateGenreRecs.js

```diff
@@ -0,0 +1,101 @@
+const postgresDb = require('../postgresDb/index.js');
+const mongoDb = require('../mongoDb/index.js');
+
+const genreArr = [
+  'action',
+  'animation',
+  'comedy',
+  'documentary',
+  'drama',
+  'family',
+  'fantasy',
+  'international',
+  'horror',
+  'musical',
+  'mystery',
+  'romance',
+  'sci_fi',
+  'thriller',
+  'western',
+];
+
+const updateGenreRecs = (genre) => {
+  postgresDb.Movie.findAll({
+    // find top 1000 movies by view count
+    order: [['views', 'DESC']],
+    limit: 1000,
+  })
+    .then(result => (
```

Comment:
Map and sort are synchronous/blocking, so you don't need to chain them as promises here. In this context, you'll probably only need to use a promise chain whenever you want to perform an asynchronous database query.

Reply:
For some reason when I was writing this code, I had problems with sort so that's why I put it in a promise.  I tried it again now and it works, so something else must have been causing the problem.  Thanks for the suggestion.